open-api/rest-catalog-open-api.yaml

```diff
@@ -1610,13 +1610,19 @@ components:
 
     PageToken:
       description:
-        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
-        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
-        signaling to the service that the response should be paginated.
-
-        Servers that support pagination will recognize `pageToken` and return a `next-page-token` in response if there are more results available.
-        After the initial request, it is expected that the value of `next-page-token` from the last response is used in the subsequent request.
-        Servers that do not support pagination will ignore `next-page-token` and return all results.
+        An opaque token that allows clients to make use of pagination for a list APIs 
+        (e.g. ListTables). Clients may initiate the first paginated request by sending an empty 
+        query parameter `pageToken` (e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`) 
+        requesting a paginated response from the server.
+
+        Servers that support pagination will identify the `pageToken` parameter and return a 
+        `next-page-token` in the response if there are more results available.  After the initial 
+        request, the value of `next-page-token` from each response is used as the parameter value 
+        for the next request.  An empty `next-page-token` in the response signals that pagination
```

Comment:
```suggestion
        for the next request.  An null `next-page-token` in the response signals that pagination
```

Reply:
I clarified the correct return values for the response.